IFC 4.3/Georeferencing_Tiny_Tin_1/UT_tiny_tin_in_MGA_56.ifc

```diff
@@ -29,7 +29,7 @@ DATA;
 #22 = IFCMAPCONVERSION(#14, #21, 0., 0., 0., 1., 0., 1.);
 #23 = IFCSITE('2_$LqFwhrElv$jrhYYj9A2', #2, 'Default Site', 'Description of Default Site', $, #15, $, $, .ELEMENT., $, $, $, $, $);
 #24 = IFCRELAGGREGATES('3iLDj9mYb82A6se0A0HaK1', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#23));
-#25 = IFCBUILDING('17MWGb$P5Dn9Ujq7lL$KyC', #2, 'tin tiny', 'Description of tin tiny', $, #15, $, $, .ELEMENT., $, $, $);
+#25 = IFCFACILITY('17MWGb$P5Dn9Ujq7lL$KyC', #2, 'tin tiny', 'Description of tin tiny', $, #15, $, $, .ELEMENT., $, $, $);
```

Comment:
`IfcFacility` [documentattion](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC1/HTML/schema/ifcproductextension/lexical/ifcfacility.htm) has only 9 attributes, and not 12 as the original `IfcBuilding`.

Reply:
Resolved in f8cf19d